x/qgb/keeper/keeper_delegate_key.go

```diff
@@ -0,0 +1,29 @@
+package keeper
+
+import (
+	"context"
+	"github.com/celestiaorg/celestia-app/x/qgb/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (k Keeper) GetDelegateKeyByOrchestrator(
+	c context.Context,
+	req *types.QueryDelegateKeysByOrchestratorAddress) (*types.QueryDelegateKeysByOrchestratorAddressResponse, error) {
+	// TODO
+	return &types.QueryDelegateKeysByOrchestratorAddressResponse{}, nil
+}
+
+// GetDelegateKeys iterates both the EthAddress and Orchestrator address indexes to produce
+// a vector of MsgSetOrchestratorAddress entires containing all the delgate keys for state
+// export / import. This may seem at first glance to be excessively complicated, why not combine
+// the EthAddress and Orchestrator address indexes and simply iterate one thing? The answer is that
+// even though we set the Eth and Orchestrator address in the same place we use them differently we
+// always go from Orchestrator address to Validator address and from validator address to Ethereum address
+// we want to keep looking up the validator address for various reasons, so a direct Orchestrator to Ethereum
+// address mapping will mean having to keep two of the same data around just to provide lookups.
+//
+// For the time being this will serve
```

Comment:
Can we make this a normal comment with only info on what the method does? If we want to discuss this further, let's do this on github.

Reply:
sure